x/projects/keeper/grpc_query_developer.go

```diff
@@ -21,5 +21,15 @@ func (k Keeper) Developer(goCtx context.Context, req *types.QueryDeveloperReques
 		return nil, err
 	}
 
-	return &types.QueryDeveloperResponse{Project: &project}, nil
+	nextEpoch, err := k.epochstorageKeeper.GetNextEpoch(ctx, uint64(ctx.BlockHeight()))
+	if err != nil {
+		return nil, err
+	}
+
+	pendingProject, err := k.GetProjectForDeveloper(ctx, req.Developer, nextEpoch)
+	if project.Equal(pendingProject) || err != nil {
```

Comment:
error should be checked first in case of project == nil

Reply:
done. see https://github.com/lavanet/lava/commit/dec5f61dbc57b5f56416130660a3dcf465410efa